request_listener/curl.go

```diff
@@ -9,10 +9,23 @@ import (
 	"strings"
 )
 
-// CurlLogDumper will log the given http.Request
-// it gonna log the request using cli curl command format
-// Caution request header and Body can be heavy!!
-// Pay attention to the dump quantity/weight
+// CurlLogDumper will log the request using the curl command format
+// /!\ Caution request header and Body can be heavy !!
+// Eg tripperware:
```

Comment:
Maybe add something like this:

`You can use this tripperware to only dump the request if `dump` HTTP Header is present` ?